test/end-to-end.js

```diff
@@ -140,6 +140,24 @@ describe('end-to-end', function() {
         done(err);
       });
     });
+
+    it('emits the `ready` event when booting is complete', function() {
+      var src = FIXTURES + '/emit.js';
+      var dest = SANDBOX + '/server/boot/emit.js'
+      fs.copySync(src, dest);
+      delete require.cache[require.resolve(SANDBOX)];
+      var app = require(SANDBOX + '/server/server');
+      expect(app.readyEventEmitted).to.be.ok();
+    });
```

Comment:
This test is not necessary, the event `booted` is triggered by `loopback-boot`. 

However, if you feel it's valuable to write an integration test capturing the contract that a server scaffolded by the template will fire `booted` event, then I am ok with that too. Just rename `ready` to `booted` everywhere.